### DISCUSSION TITLE
Changing environment variables after build step possible?

### DISCUSSION THREAD

**Discussion by mobsean** [quasarframework/quasar#12024] [CLI - PWA mode]
Hi,
I´m running a Quasar App in Kubernetes. 
For that im using docker with a build step and then I copy the /dist/pwa folder in the production nginx /usr/share/nginx/html folder.

I´m curious, I want to run that container with different environment variables (other GraphQL API etc.) and I want to inject the env vars though an k8s configMap.

Is that possible without rebuilding the whole quasar app everytime the container starts?

Thanks a lot in advance.

**Comment by mobsean** [comment]
So my workaround is that I run different parallel build chains with different env variables. 
Maybe any better ideas?

**Reply by hawkeye64** [reply]
You can send the data via a JSON file, but if it is private data, then continue doing what you are doing.
